extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -55,6 +55,7 @@ tags: {  # <amp-video>
   }
   attrs: { name: "placeholder" }
   attrs: { name: "poster" }
+  attrs: { name: "crossorigin" }
```

Comment:
Cool, I saw some other `attrs` like that, but the `<link>` tag also allows any value for its `crossorigin` attribute. Ali mentioned that we may want to allow any value for future versions of the spec